Src/CSharpier/SyntaxPrinter/SyntaxNodePrinters/InitializerExpression.cs

```diff
@@ -21,10 +21,20 @@ string groupId
             return Print(node, groupId);
         }
 
-        private static Doc Print(InitializerExpressionSyntax node, string? groupId)
+        public static Doc PrintWithLine(InitializerExpressionSyntax node)
```

Comment:
**suggestion**: I think stuff like `PrintWithLine` and `PrintWithConditionalSpace` might be better placed inlined at the call site rather than have special functions for one-off use cases.

Reply:
I was able to move the logic for `PrintWithLine` into `InitializerExpression` itself.
And because `PrintWithConditionalSpace` ended up not using the `Doc.Group`, the logic for that could be inlined in `ObjectCreationExpression`.
Much nicer this way!

Reply:
![image](https://user-images.githubusercontent.com/30291673/125166911-2498ad00-e19e-11eb-9193-302393e11adb.png)